lib/config.js

```diff
@@ -268,6 +267,7 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
     env = this.addPythonPathToEnv(env, path.join(this.srcDir, 'brave', 'chromium_src', 'python_modules'))
     env = this.addPythonPathToEnv(env, path.join(this.srcDir, 'brave', 'script'))
     env = this.addPythonPathToEnv(env, path.join(this.srcDir, 'tools', 'grit', 'grit', 'extern'))
+    if (this.sccache) env.CC_WRAPPER = 'sccache'
```

Comment:
this is already in there https://github.com/brave/brave-browser/blob/master/lib/config.js#L277